collectors/euctr/parser.py

```diff
@@ -52,10 +52,12 @@ def parse_record(res):
     data.update({key: value})
 
     # Trial results URL
+
+    data['trial_results_url'] = None
```

Comment:
Do you need to add this explicitly here? I mean, if `value` is falsy, it won't be set in the lines below as far as I can see...